packages/config/src/Config.ts

```diff
@@ -319,6 +319,29 @@ export class Config {
         return `${this.mApp}.schema.json`;
     }
 
+    // _______________________________________________________________________
+    /**
+     * Schema file path used by the active layer
+     */
+    public getSchemaInfo(): { original: string, resolved: string, local: boolean } {
+        const layer = this.layerActive();
+        const schemaUri = layer.properties.$schema ?? null;
+        if (schemaUri == null) {
+            return null;
+        }
+
+        const schemaFilePath = path.resolve(
+            schemaUri.startsWith("file://") ? fileURLToPath(schemaUri) :
+                schemaUri.startsWith("./") ? path.join(path.dirname(layer.path), schemaUri) : schemaUri);
+        const isSchemaLocal = fs.existsSync(schemaFilePath) || path.isAbsolute(schemaUri);
```

Comment:
Checking to see if the path.isAbsolute may be redundant. 
Will find out when writing unit tests 😋